src/mapgen_functions.cpp

```diff
@@ -605,12 +588,12 @@ void mapgen_forest_general(map *m, oter_id terrain_type, mapgendata dat, int tur
 
     //1-2 per overmap, very bad day for low level characters
     if (one_in(10000)) {
-        m->add_spawn(mon_jabberwock, 1, SEEX, SEEY); // fixme add to monster_group?
+        m->place_spawns( mongroup_id( "GROUP_JABBERWOCK_ONLY" ), 1, 0, 0, SEEX, SEEY, 1 );
```

Comment:
The chance of those spawns is still hard coded, shouldn't that be part of the monster group?

And I think you changed the location of the spawn slight: previously it was in the middle (at 12,12) of the 24x24 map that is used for mapgen. Now it's somewhere in the upper left 12x12 area. But this is probably not important as the monsters will move around anyway.

Reply:
> The chance of those spawns is still hard coded, shouldn't that be part of the monster group?

Hmm, I guess it could be changed to: `m->place_spawns( mongroup_id( "GROUP_JABBERWOCK_ONLY" ), 1000, 0, 0, SEEX, SEEY, 1 );`

Most of this will eventually moved from cpp to json.

> And I think you changed the location of the spawn slight: previously it was in the middle (at 12,12) of the 24x24 map that is used for mapgen. Now it's somewhere in the upper left 12x12 area. But this is probably not important as the monsters will move around anyway.

Yes, it should probably be changed to `0, 0, SEEX * 2 - 1, SEEY * 2 -1`.

Will address that in separate PR.